editor/js/libs/ui.js

```diff
@@ -377,9 +377,9 @@ UI.Select = function () {
 UI.Select.prototype = Object.create( UI.Element.prototype );
 UI.Select.prototype.constructor = UI.Select;
 
-UI.Select.prototype.setMultiple = function ( boolean ) {
```

Comment:
Would `bool` work, instead?

Reply:
I think so, yes.

Reply:
I would replace `boolean` with `bool` so it is clear what is expected.

Reply:
Agreed.